ai21_tokenizer/base_tokenizer.py

```diff
@@ -76,3 +76,78 @@ def vocab_size(self) -> int:
             int: The size of the vocabs.
         """
         pass
+
+
+class AsyncBaseTokenizer(ABC):
+    """
+    Base class for tokenizers.
+
+    This class defines the interface for tokenization operations such as encoding, decoding,
```

Comment:
Fix to - "This class defines the interface for tokenization operation" -> "This class defines the interface for async tokenization operation"